Sources/HaCWebsiteLib/routes.swift

```diff
@@ -39,6 +40,9 @@ func getWebsiteRouter() -> Router {
 
   router.all("/", middleware: NotFoundMiddleware())
 
+  // Enable http gzip compression globally
+  router.all(middleware: Compression())
```

Comment:
Double checking that we want compression to be at the end of the middleware chain and not the beginning?

Might be worth adding a comment to that effect so people don't add things below it.

Reply:
@eliotlim Any follow up on this?

Reply:
I have experimented with placing the compression middleware at various points in the middleware chain. Placing it before some of the middleware seems to cause page rendering to break, and placing it after most of the middleware (for instance, right after the `NotFoundMiddleware`) has no visible impact on the page load or content encoding in transit.

In the interest of best practice, I suppose placing it at the end of the middleware chain makes the most sense since compressing the content should occur transparently and be one of the last transformations prior to transmission. It also makes adding router middleware / routes in the future less likely to break things.

Perhaps as suggested, a comment could be added to explain compression being at the end of the middleware chain.

Reply:
Sounds good to me. Wanna get that comment in and resolve conflicts? then this will be good to go!